### ISSUE TITLE
glance_image ignores region_name specified

### ISSUE THREAD

**Issue by claytono** [ansible/ansible-modules-core#743]
The glance_image module accepts region_name as a parameter, but doesn't pass it along to the glance client when creating images.

**Comment by ansibot** [comment]
@emonty, @shrews, @juliakreger, @j2sol, @rcarrillocruz, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by rcarrillocruz** [comment]
I'd say this is not just an issue on glance_image, but on all OpenStack modules.

This will need a major change in the code base.

**Comment by emonty** [comment]
I recommend moving to os_image - glance_image is fundamentally broken in many many ways and is deprecated.

wontfix
